docs/developer/checkout.mdx

```diff
@@ -39,6 +39,176 @@ Depending on the chosen channel, the user will have access to different objects.
 
 If the checkout permission is assigned to the user, only this user and staff users with permission `MANAGE_CHECKOUTS` can query its data. Checkouts without assigned users can be queried modified using the token without additional permissions.
 
+## Address Validation
+The checkout's mutations that accepts an address as an input have field that can turn off address validation. This allows to assign partial or not fully valid address to the checkout. Providing country code is mandatory for all addresses regardless of the rules provided in this input.
+
+The address [validation](developer/api-reference/inputs/checkout-address-validation-rules.mdx) input has two boolean fields: 
+- `checkRequiredFields` - Determines if an error should be raised when the provided address doesn't have all the required fields. Default set to true.
+- `checkFieldsFormat` - Determines if an error should be raised when the provided address doesn't match the expected format. Default set to true.
+- `enableFieldsNormalization` - Determines if Saleor should apply normalization on address fields. Example: converting city field to uppercase letters. Default set to true.
+
+#### checkoutCreate
+The [`checkoutCreate`](developer/api-reference/mutations/checkout-create.mdx) mutation has optional input for providing shipping and billing addresses. By disabling an address validation there is a possibility to provide only part of an address during checkout creation. 
```

Comment:
```suggestion
The mutation has an optional input for providing shipping and billing addresses. If you want to provide only a part of the address, you can disable the address validation.
```

Reply:
@peelar  Wouldn't be good to leave api-reference here? Something like:
`The [mutation]((developer/api-reference/mutations/checkout-create.mdx)) has an optional...`

Reply:
Ah shit! Sorry, I removed it by accident. It made Grammarly go crazy 😵. Fixing it

Reply:
Ah shit! Sorry, I removed it temporarily because it made Grammarly go crazy but I forgot to switch it back. Fixed!